vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -362,6 +367,19 @@ func (d *VolumeDriver) MountVolume(name string, fstype string, id string, isRead
 
 // private function that does the job of mounting volume in conjunction with refcounting
 func (d *VolumeDriver) processMount(r volume.MountRequest) volume.Response {
+	// get volume metadata
+	status, err := d.GetVolume(r.Name)
```

Comment:
In mount, we need GetVolume always, since the volume meta data is used below for fstype and access type.